src/shared/message_handler.js

```diff
@@ -12,7 +12,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-/* eslint no-var: error, prefer-const: error */
```

Comment:
Is the removal of `prefer-const` intended?

Reply:
Yes, since it's enabled globally now; please see https://github.com/mozilla/pdf.js/blob/88f72d6b1c37dfc0dbb6715685bf8cb4ae516cd7/.eslintrc#L190